igor-web/config/igor.yml

```diff
@@ -7,3 +7,11 @@ spinnaker:
     pollInterval: 30
     
 endpoints.health.sensitive: false
+
+#travis:
+#  enabled: true
+#  masters:
+#    - name: travis-ci
+#      baseUrl: https://travis-ci.org
+#      address: https://api.travis-ci.org
+#      githubToken: XXXXXX
```

Comment:
the token is how travis authenticates https://docs.travis-ci.com/api#authentication